mw/experiments.d.ts

```diff
@@ -0,0 +1,30 @@
+interface Experiment {
+    /** 
+     * The name of the experiment
+     */
+    name: string;
+    /** 
+     * Whether the experiment is enabled
+     */
+    enabled: boolean;
+    /** 
+     * An object consisting of the experiment's buckets ("control" and at least one bucket) and their probabilities (a number < 1, eg 0.25)
+     */
+    buckets: object

```

Comment:
```suggestion
    buckets: Record<string, number>;
```

`object` type can be better described